web_list_html_widget/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please shorten headers as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py